routes/repos.js

```diff
@@ -3,14 +3,20 @@ var router = express.Router();
 var Repo = require('../models').Repo;
 var auth = require("../middleware/auth")();
 const { upgradeProject } = require('../utils/upgradeCodeJson');
+const _ = require('lodash');
+
+function parsed(entity) {
+  return JSON.parse(entity) || [];
+}
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
```

Comment:
Not part of the review for this PR but why are we using the middleware signature when we are just sending a response?

Reply:
Good point, I'll change that.